woocommerce/woocommerce
# Add tests for analytics (orders)

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Introduces a number of tests for WooCommerce > Analytics.

Closes:

https://github.com/woocommerce/woocommerce-quality/issues/626
https://github.com/woocommerce/woocommerce-quality/issues/627
https://github.com/woocommerce/woocommerce-quality/issues/628
https://github.com/woocommerce/woocommerce-quality/issues/629
https://github.com/woocommerce/woocommerce-quality/issues/625
https://github.com/woocommerce/woocommerce-quality/issues/631
https://github.com/woocommerce/woocommerce-quality/issues/630



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure e2e tests pass on CI
2. Pull branch, `pnpm env:restart`
3. `pnpm test:e2e-pw admin-analytics/analytics-data.spec.js`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
